modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/GridDhtTxFinishFuture.java

```diff
@@ -500,8 +507,11 @@ private boolean finish(boolean commit,
             }
             catch (IgniteCheckedException e) {
                 // Fail the whole thing.
-                if (e instanceof ClusterTopologyCheckedException)
+                if (e instanceof ClusterTopologyCheckedException) {
+                    log.error("Error sending message to remote node [node=" + n + ']', e);
```

Comment:
We don't log (at least not in all cases) ClusterTopologyCheckedException. Why you've decided to all logging here?
See throwing IgniteCheckedException from within `org.apache.ignite.internal.managers.communication.GridIoManager#send` for an example of not logging ClusterTopologyCheckedException.

Reply:
I’ll clean up this mistake.